### DISCUSSION TITLE
miimum version of moonraker

### DISCUSSION THREAD

**Discussion by leadustin** [Donkie/Spoolman#234] [General]
Hello
Which version of Moonraker is required as a minimum?

**Comment by rufinus** [comment]
as moonraker doesnt have a "version" per se, it is not easy to say it is in the 0.8.0 end of june 2023.
https://github.com/Arksine/moonraker/commit/4b9c836743fe0690ed60a60c9bb65587f256ea1a

**Reply by leadustin** [reply]
I am trying to install Spoolman on my Qidi Xplus-3. I get an error message: Unparsed config section [spoolman] detected. This may be the result of a component that failed to load. In the future this will result in a startup error.

I have successfully integrated both macros. Both are also displayed in Fluidd. The moonraker.conf is also configured accordingly
The Moonraker version is v0.7.1-609-gbdd0222-dirty
Fluidd and Mainsail are up to date

**Reply by rufinus** [reply]
yes 0.7.1 is way to old.

**Reply by leadustin** [reply]
That's what I was afraid of

**Reply by rufinus** [reply]
why dont you update it?

**Reply by leadustin** [reply]
The problem is that Qidi uses its own version of Klipper and Moonraker on the printer. If I now update this via KIAUH, I destroy the printer

**Reply by Donkie** [reply]
Not much we can do then. You need latest Moonraker to make it interact with Spoolman. Take this up with Qidi instead.
